src/test/scala/wdl4s/SyntaxErrorSpec.scala

```diff
@@ -256,5 +256,17 @@ class SyntaxErrorSpec extends FlatSpec with Matchers {
         |}
       """.stripMargin)
   }
+  it should "detect when expressions within a command block don't reference actual variables" in {
+    expectError(
+      """task a {
+        |  Int x
+        |  command { ./script ${x+y} }
+        |}
+        |
+        |workflow w {
+        |  call a
```

Comment:
From a test-pedantry point of view, this could just be throwing an error because "x" is never set. To be exact, make sure you say `call a {input: x=5 }` or equivalent